plug.vim

```diff
@@ -798,7 +798,7 @@ function! s:update_impl(pull, force, args) abort
     echohl None
   endif
 
-  let python = (has('python') || has('python3')) && !s:is_win && !has('win32unix')
+  let python = (has('python') || has('python3'))
       \ && (!s:nvim || has('vim_starting'))
   let ruby = has('ruby') && !s:nvim && (v:version >= 703 || v:version == 702 && has('patch374'))
```

Comment:
Since your fix only applies to the python installer, we may need to disable the ruby version on windows (which can take precedence over python when both are available).